src/azure-cli/azure/cli/command_modules/servicebus/_params.py

```diff
@@ -238,9 +238,9 @@ def load_arguments_sb(self, _):
     for scope in ['servicebus namespace network-rule add', 'servicebus namespace network-rule remove']:
         with self.argument_context(scope) as c:
             c.argument('subnet', arg_group='Virtual Network Rule', options_list=['--subnet'], help='Name or ID of subnet. If name is supplied, `--vnet-name` must be supplied.')
-            c.argument('ip_mask', arg_group='IP Address Rule', options_list=['--ip-address'], help='IPv4 address or CIDR range.')
+            c.argument('ip_mask', arg_group='IP Address Rule', options_list=['--ip-address'], help='IPv4 address or CIDR range.', deprecate_info=c.deprecate(redirect='--ip-rule', expiration='2.50.0'))
```

Comment:
```suggestion
            c.argument('ip_mask', arg_group='IP Address Rule', options_list=['--ip-address'], help='IPv4 address or CIDR range.', deprecate_info=c.deprecate(redirect='--ip-rule', expiration='2.49.0'))
```
Pls double confirm the breaking change version

Reply:
Last release was `2.47.0`............